[2022-08-11 16:45:59,865] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2022-01-02T06:00:00+00:00 [queued]>
[2022-08-11 16:45:59,909] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2022-01-02T06:00:00+00:00 [queued]>
[2022-08-11 16:45:59,911] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 16:45:59,913] {taskinstance.py:1239} INFO - Starting attempt 4 of 4
[2022-08-11 16:45:59,915] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 16:45:59,961] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-01-02 06:00:00+00:00
[2022-08-11 16:45:59,966] {standard_task_runner.py:52} INFO - Started process 3748 to run task
[2022-08-11 16:45:59,974] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2022-01-02T06:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpjn53fdc1', '--error-file', '/tmp/tmpdsmtgl19']
[2022-08-11 16:45:59,977] {standard_task_runner.py:77} INFO - Job 34: Subtask ingest
[2022-08-11 16:46:00,122] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2022-01-02T06:00:00+00:00 [running]> on host cdc85d7e6248
[2022-08-11 16:46:00,276] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-02T06:00:00+00:00
[2022-08-11 16:46:00,280] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-11 16:46:00,282] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/*** && ls']
[2022-08-11 16:46:00,299] {subprocess.py:85} INFO - Output:
[2022-08-11 16:46:00,376] {subprocess.py:89} INFO - ***.cfg
[2022-08-11 16:46:00,378] {subprocess.py:89} INFO - dags
[2022-08-11 16:46:00,380] {subprocess.py:89} INFO - logs
[2022-08-11 16:46:00,382] {subprocess.py:89} INFO - output
[2022-08-11 16:46:00,383] {subprocess.py:89} INFO - output.gz
[2022-08-11 16:46:00,386] {subprocess.py:89} INFO - plugins
[2022-08-11 16:46:00,388] {subprocess.py:89} INFO - requirements.txt
[2022-08-11 16:46:00,389] {subprocess.py:89} INFO - test
[2022-08-11 16:46:00,390] {subprocess.py:89} INFO - webserver_config.py
[2022-08-11 16:46:00,392] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-11 16:46:00,455] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20220102T060000, start_date=20220811T164559, end_date=20220811T164600
[2022-08-11 16:46:00,507] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-11 16:46:00,625] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
